model_analyzer/result/model_config_measurement.py

```diff
@@ -133,7 +133,7 @@ def get_metric(self, tag):
             return self._non_gpu_data_from_tag[tag]
         else:
             logger.warning(
-                f"No metric corresponding to tag '{tag}' "
+                f"No non-GPU metric corresponding to tag '{tag}' "
```

Comment:
Does this function only handle non-GPU because the GPU metrics are held at a higher level class (RunConfigMeasurement)?

Reply:
Yes.